daemon/ZilliqaDaemon.cpp

```diff
@@ -323,28 +411,23 @@ int main() {
 
   umask(0);
 
-  sid = setsid();
-
-  if (sid < 0) {
+  if (setsid() < 0) {
     log << "Unable to set sid" << endl;
     exit(EXIT_FAILURE);
   }
 
   if ((chdir("..")) < 0) {
-    log << "Failed to chdir" << endl;
```

Comment:
seems this logging is dropped accidentally. I couldn't get it why changing to the parent directory is needed. Is there any assumption about the working directory where the daemon is executing?